bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -389,8 +389,11 @@ Intercept of free living Nitrogen fixation with zero annual ET
 </entry>
 
 <entry id="nfix_method" type="char*25" category="cnfun_inparm"
-       group="cnfun_inparm" valid_values="Houlton,Bytnerowicz" value="Houlton" >
+       group="cnfun_inparm" valid_values="Houlton,Bytnerowicz" value="Bytnerowicz" >
```

Comment:
It's not a problem to update the default value, but it shouldn't matter, because the default is explicit in the namelist_defaults file.

Reply:
Since it doesn't matter, I will leave this one the way that I changed it, because it helps makes it more obvious (at least in my mind) that this is the current default.